tests/AttributesTest.php

```diff
@@ -98,7 +98,7 @@ public function testShapeWithExtraKey(): void {
       $x =
         <test:attribute-types
           /* HH_IGNORE_ERROR[4110] */
-          /* HH_IGNORE_ERROR[4166] */
+          /* HH_IGNORE_ERROR[4166] This supression is not needed under HHVM 4.14, but needed in lower versions. */
```

Comment:
Your comment isn't any less clear than it was before, but I recommend making suppression comments explain the error (e.g. "HHVM < 4.14 doesn't handle shape types correctly" or something)

Also I think we aim for 80 character lines.

Reply:
Understood